storage.js

```diff
@@ -0,0 +1,29 @@
+const buttonSubmit = document.querySelector('.Collaboration');
+const nameId = document.getElementById('name');
+const emailId = document.getElementById('email');
+const commentId = document.getElementById('comment-box');
+
+const formDetails = JSON.parse(localStorage.getItem('contactInfo'));
+if (formDetails) {
+  nameId.value = formDetails.name;
+  emailId.value = formDetails.email;
+  commentId.value = formDetails.comment;
+}
+
+buttonSubmit.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const name = nameId.value;
+  const email = emailId.value;
+  const comment = commentId.value;
+
+  if (!name || !email || !comment) {
+    return;
+  }
+  // Store data on array//
+  const contactInfo = {
+    name,
+    email,
+    comment,
+  };
+  localStorage.setItem('contactInfo', JSON.stringify(contactInfo));
+});
```

Comment:
- [ ] Nice work using `localStorage.getItem` and `localStorage.setItem` funciton to save and retrieve the form data object :clap:  However form information should be saved and updated any time the user changes the content of any input field in the form, but your code is saving the form when user clicks the submit buttons. Kindly check it and make sure form information is persistent to page reloads